process/slash/constants.go

```diff
@@ -1,17 +1,20 @@
 package slash
 
-type SlashingType string
+import coreSlash "github.com/ElrondNetwork/elrond-go-core/data/slash"
 
+// Used by slashing notifier to create a slashing transaction
+// from a proof. Each transaction identifies a different
+// slashing event based on this ID
 const (
-	None             SlashingType = "no slashing"
-	MultipleProposal SlashingType = "multiple header proposal"
-	MultipleSigning  SlashingType = "multiple header signing"
+	// MultipleProposalProofID = MultipleProposal's ID
+	MultipleProposalProofID byte = 0x1
```

Comment:
these are also declared in the elrond-go-core, could they be used from there?

Reply:
Yes. Each PR in `elrond-go` references a certain PR in `elrond-go-core`.
In the next PR (https://github.com/ElrondNetwork/elrond-go/pull/3596) which is going to be merged in this one, they are removed